source/developer/developer-setup.md

```diff
@@ -90,6 +90,14 @@ Any issues? Please let us know on our forums at: http://forum.mattermost.org
 
 ### Archlinux ###
 
+#### Build and install with AUR ####
+
+Archlinux uses a specific feature called [Arch User Repository](https://wiki.archlinux.org/index.php/Arch_User_Repository) to manage extra packages which do not belong to the official repository. It allows you to compile a package from source and then install it via the Arch package manager [pacman](https://wiki.archlinux.org/index.php/Pacman#Additional_commands). This way you will be able to track easily upgrades or dependency issues. This is the recommended way to install extra packages.
```

Comment:
Since this is unofficial can we please remove this sentence: `This is the recommended way to install extra packages.`